test/compilable/test17373.d

```diff
@@ -17,4 +18,15 @@ interface Simple
 }
 static assert(__traits(getOverloads, Simple, "square").length == 2);
 
-void main() {}
+// https://issues.dlang.org/show_bug.cgi?id=19064
+interface InputStream {}
+interface OutputStream{}
+interface Stream : InputStream, OutputStream{}
+interface ConnectionStream : Stream
+{
+    @property bool connected() const;
+    void close();
+}
+
+static assert(__traits(getOverloads, ConnectionStream, "connected").length == 1);
+static assert(__traits(getOverloads, ConnectionStream, "close").length == 1);
```

Comment:
Why not `.stringof` and do a full comparison?

Reply:
It's the same thing.

Reply:
Yes, but it ensures that you don't accidentally return the wrong one. Let's better be safe than sorry.
I added this as myself and hope you don't mind?